examples/06-multiple-connections/client.c

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2020, Intel Corporation */
+
+/*
+ * client.c -- a client of the multiple-connections example
+ *
+ * Please see README.md for a detailed description of this example.
+ */
+
+#include <librpma.h>
+#include <inttypes.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#include "common.h"
+
+static char *Names[] = {
+	"Oksana",
+	"Lukasz",
+	"Tomasz",
+	"Janek",
+	"Chet"
+};
+
+#define NAMES_NUM (sizeof(Names) / sizeof(Names[0]))
+
+#define USAGE_STR "usage: %s <server_address> <service>\n"
+
+int
+main(int argc, char *argv[])
+{
+	/* validate parameters */
+	if (argc < 3) {
+		fprintf(stderr, USAGE_STR, argv[0]);
+		exit(-1);
+	}
+
+	/* read common parameters */
+	char *addr = argv[1];
+	char *service = argv[2];
+	int ret, ret2;
+
+	/* RPMA resources - general */
+	struct rpma_peer *peer = NULL;
+	struct rpma_conn *conn = NULL;
+
+	/* resources - memory region */
+	void *mr_ptr = NULL;
+	struct rpma_mr_local *mr = NULL;
+
+	/*
+	 * lookup an ibv_context via the address and create a new peer using it
+	 */
+	ret = client_peer_via_address(addr, &peer);
+	if (ret)
+		return ret;
+
+	/* allocate a memory */
+	mr_ptr = malloc_aligned(DWORD);
```

Comment:
what's 'DWORD'? 4? 8?

This is a fairly cryptic. I'd prefer `sizeof(void *)` or `sizeof(uintptr_t)` if you want to have a size of pointer.